genkeys.go

```diff
@@ -130,12 +134,16 @@ func init() {
 		0xc0: "GraveAccent",
 		0x08: "Backspace",
 		0x90: "NumLock",
+		0x91: "ScrollLock",
 		0x6e: "KPDecimal",
 		0x6f: "KPDivide",
 		0x6a: "KPMultiply",
 		0x6d: "KPSubtract",
 		0x6b: "KPAdd",
 
+		// On Chrome and Edge, this does not work seem to be captured. At least on Windows.
+		//0x2C: "PrintScreen",
```

Comment:
You don't have to comment this out. This should not be harmful. I'll check this later.